.github/chatmodes/PAW-01A Spec Agent.chatmode.md

```diff
@@ -306,6 +322,21 @@ Use this during validation (auto-generate in narrative, not as a committed file
 - [ ] Each story has ≥1 acceptance scenario
 - [ ] Edge cases enumerated
 
+### Narrative Quality
+- [ ] Overview section exists with 2-4 paragraphs of narrative prose
+- [ ] Overview focuses on WHAT the feature does and WHY it matters from user perspective
+- [ ] Overview avoids implementation details (no file paths, architecture, code references)
+- [ ] Overview uses specific, measurable language (not vague terms like "fast" or "user-friendly" without quantification)
+- [ ] Overview maintains clear, flowing prose that tells a coherent story (not bullet fragments)
+- [ ] Overview does not mix in formal specifications, schemas, or code-like elements
+- [ ] Objectives section exists with bulleted list of behavioral goals
+- [ ] Objectives describe observable outcomes, not technical approaches
+- [ ] Objectives are technology-agnostic and focus on WHAT, not HOW
+- [ ] Objectives may include optional rationale notes explaining the why behind each goal
+- [ ] Non-functional requirements (if any) are quantified with specific metrics or thresholds
+- [ ] Narrative sections do not duplicate User Stories, FRs, or Success Criteria content
+- [ ] Overview narrative logically connects to and sets up the User Stories section that follows
```

Comment:
This feels like too many points to dedicate to the narrative section; let's consolidate and aim for ~4 high impact quality checks.